forge-gui/res/cardsfolder/a/anduril_flame_of_the_west.txt

```diff
@@ -6,7 +6,7 @@ T:Mode$ Attacks | ValidCard$ Card.AttachedBy | Execute$ TrigBranch | TriggerDesc
 SVar:TrigBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ EQ1 | TrueSubAbility$ Legendary | FalseSubAbility$ NonLegendary
 SVar:Legendary:DB$ Token | TokenAmount$ 2 | TokenScript$ w_1_1_spirit_flying | TokenOwner$ You | TokenTapped$ True | TokenAttacking$ True
 SVar:NonLegendary:DB$ Token | TokenAmount$ 2 | TokenScript$ w_1_1_spirit_flying | TokenOwner$ You | TokenTapped$ True
-SVar:X:Count$Valid Creature.EquippedBy+Legendary
+SVar:X:Count$Valid Creature.TriggedAttacker+Legendary
```

Comment:
yea good call
but actually you should use `TriggeredAttacker$Valid Card.Legendary`
because even when the creature gets removed in response LKI should take over instead